packages/react-components/react-rating-preview/stories/Rating/RatingColor.stories.tsx

```diff
@@ -14,19 +14,19 @@ export const Color = () => {
   const styles = useStyles();
   return (
     <div className={styles.root}>
-      <Rating color="brand" defaultValue={3} ratingLabel={3} ratingCountLabel="1,160" />
+      <Rating defaultValue={3.5} />
 
-      <Rating color="marigold" defaultValue={3.5} ratingLabel={3.5} ratingCountLabel="1,160" />
+      <Rating color="brand" defaultValue={3} />
 
-      <Rating defaultValue={3.5} ratingLabel={3.5} ratingCountLabel="1,160" />
+      <Rating color="marigold" defaultValue={3.5} />
```

Comment:
The 3.5 default value doesn't work unless step is set right? probably better to focus this story only on color, and set `defaultValue={3}` for all of them.